cmd/account/account.go

```diff
@@ -2,17 +2,27 @@ package account
 
 import (
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
 
 	"github.com/oasisprotocol/cli/cmd/account/show"
 )
 
-var Cmd = &cobra.Command{
-	Use:     "account",
-	Short:   "Account operations",
-	Aliases: []string{"a", "acc", "accounts"},
-}
+// SubtractFeeFlags is a force mode switch.
```

Comment:
This comment seems to be in the wrong place (it covers the whole block) and also I have no idea what it means. Also, should the flag move under `common` so it could in theory be used outside `account`?